jenkins/custom-ci/centos-7.9.2009/motr.groovy

```diff
@@ -17,6 +17,7 @@ pipeline {
 	parameters {  
         string(name: 'MOTR_URL', defaultValue: 'https://github.com/Seagate/cortx-motr', description: 'Branch for Motr.')
 		string(name: 'MOTR_BRANCH', defaultValue: 'stable', description: 'Branch for Motr.')
+		string(name: 'MOTR_BUILD_MODE', defaultValue: 'user-mode', description: 'Build motr rpm using kernel or user-mode.', trim: true)
```

Comment:
can't we keep this var in choice format here as well?

Reply:
added new changes and tested with  - http://eos-jenkins.colo.seagate.com/job/Release_Engineering/job/re-workspace/job/custom-ci-test-EOS-25657/110/console